docs/pages/blog/2020-q1-update.md

```diff
@@ -0,0 +1,110 @@
+---
+description: An update on our mission for Q1 2020.
+---
+
+# Q1 2020 Update
+
+Olivier Tassinari. April 14, 2020.
+
+Welcome to the new format of our mission update. We are moving from monthly to quarterly updates.
+
+## Product
+
+Over the last 3 months, we have focused on making patch releases.
+We have done eleven so far. We have optimized for stability.
+
+Here are the most significant improvements since the beginning of the year:
+
+- 🔍 We have polished the [Autocomplete](https://material-ui.com/components/autocomplete/) component (Combo box). We have closed more than [270](https://github.com/mui-org/material-ui/labels/lab%3A%20Autocomplete) issues and pull requests so far. We will promote the component to the core in the next major (v5).
+
+  ![autocomplete](/static/blog/2020-q1-update/autocomplete.gif)
+
+  <p class="blog-description">useAutocomplete <a href="https://material-ui.com/components/autocomplete/#useautocomplete">hook</a> example, 4.5 kB gzipped.</p>
+
+  If you wish to make your first contribution to open source, the component has a couple of ["good first issues"](https://github.com/mui-org/material-ui/labels/lab%3A%20Autocomplete) that can be taken.
+  If you have TypeScript expertise, the component could [benefit from it](https://github.com/mui-org/material-ui/issues?q=is%3Aopen+label%3A%22lab%3A+Autocomplete%22+label%3Atypescript).
+
+- 📆 We have made a first [alpha release](https://next.material-ui-pickers.dev/demo/daterangepicker) of the date range picker. It's not ready to be used in production but you can start playing. We might release some of the new features of the date picker under the upcoming Enterprise offering.
+
+  ![date picker](/static/blog/2020-q1-update/date-picker.png)
+
+- ⚠️ We have introduced a new [Alert](https://material-ui.com/components/alert/) component in the lab. While this component isn't mentioned in the Material Design specification, it's a common and well-established pattern. For instance, it can be interested when [combined](https://material-ui.com/components/snackbars/#notistack) with the Snackbar.
+
+  <img src="/static/blog/2020-q1-update/alert.png" alt="alert" style="max-height: 369px; margin-bottom: 24px;" />
+
+- ⏭ We have introduced a new [Pagination](https://material-ui.com/components/pagination/) component.
+
+  <img src="/static/blog/2020-q1-update/pagination.png" alt="pagination" style="max-height: 208px; margin-bottom: 8px;" />
+
+- 🦴 We have added a new "wave" animation to the [Skeleton](https://material-ui.com/components/skeleton/#animations) component.
+
+  <video autoplay muted loop style="max-height: 95px; margin-bottom: 24px;">
+    <source src="/static/blog/2020-q1-update/skeleton.webm" type="video/webm" />
+  </video>
+
+- ⚛️ We have worked on improving the developer experience inside the text editors.
```

Comment:
```suggestion
- ⚛️ We have worked on improving developer experience inside text editors.
```
I think it reads nicer without 'the' cc @mbrookes

Reply:
Too late...